src/lucky_migrator/runner.cr

```diff
@@ -33,6 +33,11 @@ class LuckyMigrator::Runner
     if (message = e.message) && message.includes?(%("#{self.db_name}" already exists))
       puts "Already created #{self.db_name.colorize(:green)}"
       exit(0)
+    elsif (message = e.message) && (message.includes?("createdb: not found") || message.includes?("No command 'createdb' found"))
+      puts "* Try installing postgres tools if you are on macOS https://postgresapp.com/documentation/cli-tools.html
```

Comment:
Could you raise here instead of puts? That way it exits if it didn’t fail. You could then remove the exit(0). 

Also, could you make it look like this arrow by using the green arrows instead of asterisks? https://github.com/luckyframework/lucky_migrator/commit/722fb00b44f6490e94c0d37be6cfff48b41374da#diff-542ad218b9c2af24cfd70b07f3aed299R23

Also, I love that you put “if you’re on Linux” first. Could ou do the same for macOS? Like “if you’re in macOS, try installing...”

Reply:
I included `LuckyCli::TextHelpers` but I couldn't access the `green_arrow` method from `self.create_db`, presumably because it's a class method. What should I do in this case?

Reply:
In that case you can use “extend” instead of “include” and you ou should be able to use it in class methods

Reply:
Got it, thanks